app/lib/actions/remote_execution/run_host_job.rb

```diff
@@ -24,11 +24,16 @@ def plan(job_invocation, host, template_invocation, proxy)
 
         raise _('Could not use any template used in the job invocation') if template_invocation.blank?
 
-        settings = { :global_proxy   => 'remote_execution_global_proxy',
-                     :fallback_proxy => 'remote_execution_fallback_proxy' }
+        if proxy.blank?
+          offline_count = options[:offline_count] || 0
+          raise N_('All %{number} usable proxies are down.') % { :number => offline_count } if offline_count > 0
```

Comment:
Oh, sorry, thought that was a typo. Should it be like this? Can't test it right now